lib/rubocop/formatter/formatter_set.rb

```diff
@@ -88,7 +89,12 @@ def builtin_formatter_class(specified_key)
           /^\[#{specified_key}\]/.match?(key) || specified_key == key.delete('[]')
         end
 
-        raise %(No formatter for "#{specified_key}") if matching_keys.empty?
+        if matching_keys.empty?
+          similar_name = NameSimilarity.find_similar_name(specified_key, BUILTIN_FORMATTER_NAMES)
+          suggestion = %( Did you mean? "#{similar_name}") if similar_name
+
+          raise Rainbow(%(No formatter for "#{specified_key}".#{suggestion})).red
```

Comment:
Maybe change this to "Formatter x not found/does not exist. ..."? The current message looks a bit off to me.

Reply:
Thanks! I've updated the message.